lib/etl/load/create_new_group_records.rb

```diff
@@ -0,0 +1,32 @@
+class CreateNewGroupRecords
+  def write(attributes_list)
```

Comment:
Should this class accept an individual set of attributes, rather than a list of attributes? I was under the impression that the ETL script should be loading one record at a time, but maybe I'm mistaken.